api/envoy/config/cluster/v3/circuit_breaker.proto

```diff
@@ -103,3 +103,13 @@ message CircuitBreakers {
   // are used.
   repeated Thresholds thresholds = 1;
 }
+
+message DnsCacheCircuitBreaker {
+  message DnsCacheThreshold {
```

Comment:
do we need this intermediate message?

Reply:
I think that this is right because we won't use this message from properties on other messages.